src/main/java/domain/LottoNumberGenerator.java

```diff
@@ -0,0 +1,44 @@
+package domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumberGenerator {
+
+	private final int startNumber;
+	private final int endNumber;
+	private final int limitNumber;
+
+	public LottoNumberGenerator(int startNumber, int endNumber, int limitNumber) {
+		this.startNumber = startNumber;
+		this.endNumber = endNumber;
+		this.limitNumber = limitNumber;
+	}
+
+	public LottoNumbers pickNumber() {
+		List<Integer> pickNumbers = IntStream.range(startNumber, endNumber)
```

Comment:
메서드명은 `pickNumber()` 인데 반환은 Numbers로 하고있네요?.?
```suggestion
	public LottoNumbers pickNumbers() {
```

Reply:
메서드명을 `generateLottoNumbers` 라고 변경했습니다.